README.rst

```diff
@@ -223,11 +372,173 @@ Configure celery (you may use a different broker if you want):
         },
     }
 
-If you decide to use redis (as shown in these examples),
-install the requierd python packages::
+    INSTALLED_APPS.append('djcelery_email')
+    EMAIL_BACKEND = 'djcelery_email.backends.CeleryEmailBackend'
+
+If you decide to use Redis (as shown in these examples),
+install the following python packages.
+
+.. code-block:: shell
 
     pip install redis django-redis
 
+Quickstart Guide
```

Comment:
This section looks really good right now. 

Maybe, we can also copy this section to the openwisp2-docs in the future.